runelite-client/src/main/java/net/runelite/client/plugins/hunter/HunterPlugin.java

```diff
@@ -117,7 +117,7 @@ public void onGameObjectSpawned(GameObjectSpawned event)
 			case ObjectID.DEADFALL: // Deadfall trap placed
 			case ObjectID.MONKEY_TRAP: // Maniacal monkey trap placed
 				// If player is right next to "object" trap assume that player placed the trap
-				if (localPlayer.getWorldLocation().distanceTo(trapLocation) <= 1)
+				if (localPlayer.getWorldLocation().distanceTo(trapLocation) <= 2)
```

Comment:
This will break maniacal monkey traps ( as they are close to each other, so other player's traps can be registered). probably special case this for non-monkey trap)